Remove doc about the removed command shouldafound (#1170)
returntocorp/semgrep-docs: docs/reporting-false-negatives.md

---

import MoreHelp from "/src/components/MoreHelp"

# Reporting false negatives

> Just submit an issue on the semgrep issue tracker here: https://github.com/returntocorp/semgrep/issues

Reviewer: ```suggestion
Submit an issue on the [Semgrep issue tracker](https://github.com/returntocorp/semgrep/issues).
```

> It's usually better to include a playground link with a rule and some test code illustrating the false negative.

Reviewer: ```suggestion
If applicable, include a [Semgrep Playground](https://semgrep.dev/playground) link with a **rule** and some **test code** illustrating the false negative.
```
